tests/consensus_tests/test_resharding_set_replica_dead_replay.py

```
@@ -0,0 +1,171 @@
+    # `Resharding` state. We then drive a resharding-stream-records transfer
+    # from an existing shard to shard 2 on a non-target peer so shard 2 has
+    # *two* `Resharding` replicas — without the second replica, the
+    # failure-detection path's "last active peer" guard refuses to propose
+    # `SetShardReplicaState(Dead)` and the deactivation never happens.
+    create_collection(peer_uris[0], collection=COLLECTION, shard_number=2, replication_factor=1)
+    wait_collection_exists_and_active_on_all_peers(COLLECTION, peer_uris)
+
+    new_shard_dir = target_dir / "storage" / "collections" / COLLECTION / "2"
+    new_replica_state = new_shard_dir / "replica_state.json"
+
+    resp = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)